pkg/infra/proposer_test.go

```diff
@@ -55,4 +58,32 @@ var _ = Describe("Proposer", func() {
 			Expect(err).Should(MatchError(ContainSubstring("error connecting to invalid_addr")))
 		})
 	})
+
+	Context("Tape should do less for prepare and summary endorsement process", func() {
```

Comment:
I still don't think we need this particular test (it doesn't provide much value as is). But one more test doesn't harm anyway.